Units/MMLAddon/LPInc/Classes/MML/lptmufasabitmap.pas

```diff
@@ -512,25 +512,25 @@ procedure Register_TMufasaBitmap(Compiler: TLapeCompiler);
     addClassVar('TMufasaBitmap', 'Height', 'Integer', @TMufasaBitmap_Height_Read);
     addGlobalFunc('procedure TMufasaBitmap.SetPersistentMemory(mem: PtrUInt; awidth, aheight: integer);', @TMufasaBitmap_SetPersistentMemory);
     addGlobalFunc('procedure TMufasaBitmap.ResetPersistentMemory();', @TMufasaBitmap_ResetPersistentMemory);
-    addGlobalFunc('function TMufasaBitmap.PointInBitmap(x,y : integer): boolean;', @TMufasaBitmap_PointInBitmap);
-    addGlobalFunc('procedure TMufasaBitmap.ValidatePoint(x,y : integer);', @TMufasaBitmap_ValidatePoint);
+    addGlobalFunc('function TMufasaBitmap.PointInBitmap(x,y : integer): boolean; constref;', @TMufasaBitmap_PointInBitmap);
+    addGlobalFunc('procedure TMufasaBitmap.ValidatePoint(x,y : integer); constref;', @TMufasaBitmap_ValidatePoint);
     addGlobalFunc('function TMufasaBitmap.SaveToFile(const FileName : string): boolean;', @TMufasaBitmap_SaveToFile);
     addGlobalFunc('procedure TMufasaBitmap.LoadFromFile(const FileName : string);', @TMufasaBitmap_LoadFromFile);
     addGlobalFunc('procedure TMufasaBitmap.Rectangle(const Box : TBox;FillCol : TColor);', @TMufasaBitmap_Rectangle);
     addGlobalFunc('procedure TMufasaBitmap.Rectangle(const Box: TBox; const Color: Integer; const Transparency: Extended); overload;', @TMufasaBitmap_RectangleEx);
     addGlobalFunc('procedure TMufasaBitmap.FloodFill(const StartPT : TPoint; const SearchCol, ReplaceCol : TColor);', @TMufasaBitmap_FloodFill);
-    addGlobalFunc('procedure TMufasaBitmap.SetPixel(x,y : integer; Color : TColor);', @TMufasaBitmap_FastSetPixel);
-    addGlobalFunc('procedure TMufasaBitmap.SetPixels(Points : TPointArray; Colors : TIntegerArray);', @TMufasaBitmap_FastSetPixels);
+    addGlobalFunc('procedure TMufasaBitmap.SetPixel(x,y : integer; Color : TColor); constref;', @TMufasaBitmap_FastSetPixel);
```

Comment:
I was very quick when going over TMufasaBitmap
"*Also added a couple to TMufasaBitmap, maybe more should be added, I only added to the most obvious methods.*"
The most obvious methods for me was the ones that I instantly recognized.